SecurityPkg/Tcg/Tcg2Dxe/Tcg2Dxe.c

```diff
@@ -2333,6 +2334,36 @@ MeasureAllSecureVariables (
     DEBUG ((DEBUG_INFO, "Skip measuring variable %s since it's deleted\n", EFI_IMAGE_SECURITY_DATABASE2));
   }
 
+  //
+  // Meaurement UEFI device signature database
+  //
+  if ((PcdGet32 (PcdTcgPfpMeasurementRevision) >= TCG_EfiSpecIDEventStruct_SPEC_ERRATA_TPM2_REV_106) &&
+      (PcdGet8 (PcdEnableSpdmDeviceAuhenticaion) != 0)) {
+    Status = GetVariable2 (EDKII_DEVICE_SECURITY_DATABASE, &gEfiDeviceSignatureDatabaseGuid, &Data, &DataSize);
+    if (Status == EFI_SUCCESS) {
+      Status = MeasureVariable (
+                PCR_INDEX_FOR_SIGNATURE_DB,
+                EV_EFI_SPDM_DEVICE_POLICY,
+                EDKII_DEVICE_SECURITY_DATABASE,
+                &gEfiDeviceSignatureDatabaseGuid,
+                Data,
+                DataSize
+                );
+      FreePool (Data);
+    } else if (Status == EFI_NOT_FOUND) {
+      Data = NULL;
+      DataSize = 0;
+      Status = MeasureVariable (
+                PCR_INDEX_FOR_SIGNATURE_DB,
+                EV_EFI_SPDM_DEVICE_POLICY,
+                EDKII_DEVICE_SECURITY_DATABASE,

```

Comment:
I think here `EDKII_DEVICE_SECURITY_DATABASE` should be changed to `EFI_DEVICE_SECURITY_DATABASE`.